.travis.yml

```diff
@@ -55,7 +57,10 @@ env:
         - PATH="$INSTALLDIR/bin:$PATH"
         - LD_LIBRARY_PATH="$INSTALLDIR/lib:$LD_LIBRARY_PATH"
         - BUILD_DOCS=false
+        - DEPLOY_DOCS=false
         - TEST=false
+        # Encrypted Github token (GH_TOKEN)
+        - secure: "cwHlTNa+b2lPiCbN5Ja9alj6EkKwJsuR5QcNPORrp3uCOQF9+cJUFEyMIGVGzrzZQnpjn05XYDr6eGFUYkbaR1QVUQdNH+2rCiScFHe94fRPJlJabxXexCfvZCNV8VRi6NSyFPuZKkCVs10bQ7Lk5dPS/V4Dalz1xyURcvI8H8XQFTmDYb/7QFw73l0WzNf+Y1rzNpRZEQ4ERhmnir489njCv1w7oJ0C+BrRvPqTmNmfXbrY6Bcana6BJvvumSY1xRaw60f/UNKYTlTVlYTVSWK1kq9mTfkhiJhe5oNoWchKloiOg3fA2XtAuucuXSdCjqo9aWHMxKJy+jQ7/BeId9hacvB9pVclYoOLARBcPoWDb21eKJZ8A70yKlUM3ryhVXM4drHzM4ITitPA1sEJWNJ2BQO8Cils0FfcOLvBR3NdbLi8RdHZItmU/Oe8o1srUTnnkdPRiEWbmAEs4pVrsHY9sD0660PXTFBSQ7sDXeyTl2etG/YSQKvE3ysOnZwhEuTXqf8cpo+tSbjindvWSyyN0HSuJ0bd7Ei+qWUQ3nJNEtHpiq8Ms6KUPHjPpWO2BEl+FQHWFpmRtwBg/REk36VWIl1UHSKuKkqxUT4qiZeL5zlrHGso+7QGhVjkYj5un3XoVObopFNVUB8y56xeW21gI51HK48cPd5rMnX95KE="
```

Comment:
@leouieda Can you access this link (https://codecov.io/gh/GenericMappingTools/gmt/settings) for codecov token? If you can,  I think it would be better to add the codecov token here.

Reply:
I can access it but I can't copy it. Is there any reason why you prefer having it here?

Reply:
I prefer adding CODECOV_TOKEN here, so that we can manage these two tokens at the same place. Currently, GH_TOKEN is in .travis.yml, while CODECOV_TOKEN in travis's web interface.

Reply:
OK. We could also move the GH_TOKEN to the web interface then. As I said, I can't add the codecov token because I don't have access to it. But if you have it, then could you add it here instead?

Reply:
@leouieda I've added encrypted CODECOV_TOKEN to travis.yml and removed it from the web interface.

Reply:
👍 thanks, Dongdong!